kadai1/takakd/internal/imgconv/imgconv.go

```diff
@@ -0,0 +1,160 @@
+// 画像変換モジュール
+package imgconv
+
+import (
+	"errors"
+	"fmt"
+	"image"
+	"image/gif"
+	"image/jpeg"
+	"image/png"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Supported image type
+const (
+	PNG = "png"
+	GIF = "gif"
+	JPG = "jpg"
+)
+
+type ImageFormat string
```

Comment:
```suggestion
type ImageFormat string

const (
	PNG ImageFormat = "png"
	GIF ImageFormat = "gif"
	JPG ImageFormat = "jpg"
)
```
せっかく作った型なので活用すると良いかも

Reply:
型作ったのに暗黙の型変換になってました……
ありがとうございます。